types/forge-viewer/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false
```

Comment:
Considering you don't change any export modifiers (anymore?) you should be able to revert this change to the `tslint.json` file.

Reply:
I keep the original format for our users, but to keep the module declaration, I have turn off this tslint rule.

Reply:
@weswigham If I use `export namespace` syntax instead, that will break lots of existing apps of our Forge viewer users.  Therefore, I have to omit this rule `strict-export-declare-modifiers` in the new commit. Should I revert the change?

Reply:
? What's this about `export namespace`? This rule is just about `declare` modifiers...? From what I can see, this PR is a one-line change to add a `static` modifier + a new test...?

Reply:
Yes, according to the contribution guide, I have to write a test file for any changes, but the tslint will throw an error if I use the module to declare the typing. 

The ’static’ one is the request form our paid customer of the Autodesk Forge platform. Without this, it breaks our customers’ app while compiling the TypeScript to Javascript. Please don't block it to make Autodesk lose customers. We cannot take any risk to lose that one. Thanks!

Reply:
A few days ago, while running the tslint, it asked to change the declaration of the forge-viewer module as below, I didn't know why.

```
// From
declare namespace Autodesk {
    namespace Viewing {
    }
}

// To
import * as THREE from 'three';

export namespace Autodesk {
    namespace Viewing {
    }
}
```

After @liskaj reviewed the PR, I found a way to revert back the `declare namespace Autodesk`, but it needed to omit the rule `strict-export-declare-modifiers` in the tslint. That's why I added the second commit. Now I amended the first commit with the changes I made in the second one. I didn't know why it passed now with omitting the rule.

Please don't block this PR, this request is from our paid Forge platform users. Autodesk cannot afford any risk to lose the customer. Thanks!

Reply:
Right, there's no issues with the changes to the types or the tests as they currently stand, I'm just asserting that I thought you've reverted the changes that required you to include this lint suppression over the course of this PR (which is borderline trivial at this point), and it does not need to be here anymore, from what I can tell, as it's not suppressing any failures (though if it did I'd have suggested a different way of stating the types to avoid the suppression). If I'm mistaken and the rule's a new addition that causes failures in the current forge types (and simply hadn't been run before), I'd prefer an inline suppression on the failing line to a global one. (Given that this is a declaration file, it's probably just requesting you drop a redundant `declare` modifier if that is the case - you needn't replace it with `export` - everything in a declaration file is implicitly `declare`'d)

Reply:
Thank you very much for sharing your opinion. It seems to me that there are lots of things to learn. I will try inline suppression next time. Cheers!